### DISCUSSION TITLE
How can I add my own filters and rules to ABP-style adblock?

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#6678] [Q&A] [answered]
uBlock Origin offers `my filters` tab and `my rules` tab.
How can I emulate `my filters` tab and `my rules` tab of uBlock Origin in qutebrowser?

uBlock Origin and multi-account containers are the only add-ons that I use on firefox.

**Comment by The-Compiler** [comment, marked as answer]
You can add a `file://` URL to `content.blocking.adblock.lists`.

**Reply by crocket** [reply]
That's good.

**Reply by crocket** [reply]
But, can python-adblock emulate `my rules` tab of uBlock Origin?

https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-rule-syntax

I want to block google.com and facebook.com on all websites or certain third party websites.

Or, can static filters(my filters tab) emulate dynamic filters(my rules tab)?

**Reply by The-Compiler** [reply]
AFAIK you [should be able to do the same thing](https://help.eyeo.com/en/adblockplus/how-to-write-filters#domain-restrictions) using normal, ABP-like, rules.
